### DISCUSSION TITLE
Export esmbuild metafile

### DISCUSSION THREAD

**Discussion by woody146** [remix-run/remix#4819] [Proposals]
Can you export esmbuild metafile to analyse bundle, please? Or show size of pages such as nextjs

![IMG](https://user-images.githubusercontent.com/1003261/90968340-0a7bb080-e4a0-11ea-8ac1-d3b436b1935e.png)

**Comment by lili21** [comment]
Right now, You need to use patch-package to modify the esbuild config to achieve this. 
Here is a patch https://github.com/kiliman/remix-esbuild-analysis

**Comment by woody146** [comment]
Great solution, but i think remix should support offically

**Comment by sergiodxa** [comment]
Duplicated of #3387
